src/test/java/com/example/amazoncom/BrowserStackSetup.java

```diff
@@ -0,0 +1,42 @@
+package com.example.amazoncom;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class BrowserStackSetup {
+  String driverPath = "C:\\geckodriver.exe";
+  WebDriver driver;
+  BrowserStackHomePage objBrowserStackHomePage;
+  BrowserStackSignUpPage objBrowserStackSignUpPage;
+  @BeforeTest
+  public void setup() {
+    System.setProperty("webdriver.chrome.driver",
+        "C:\\Users\\Julio_Morales\\chromedriver\\chromedriver.exe");
+    driver = new ChromeDriver();
+    driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+    driver.get("https://www.amazon.es/");
+  }
+
+  @Test(priority = 1)
+  public void navigate_to_homepage_click_on_getstarted() {
```

Comment:
use camel case pls in accordance with clean code practices described in your homework learn materials